DEPLOYMENT.md

```diff
@@ -1,5 +1,56 @@
-Follow https://central.sonatype.org/pages/apache-maven.html
+# Deploying of new release version
 
+To release a new version, you need manually publish it to maven repository (Maven Central). This requires the following steps:
+
+1. Set up and configure GPG key for generating a signature. See this [guide](https://central.sonatype.org/pages/working-with-pgp-signatures.html) 
+for more information.
+   2. Install GnuPG or jump to the next point. 
```

Comment:
Minor: These nested lists get formatted weirdly on GH. Could you try to tweak it a bit? 🙂 Would be a pity to have such a nice doc formatted improperly 🙃

Reply:
Fixed, thanks 👍